packages/navigator/src/browser/fuzzy-search.spec.ts

```diff
@@ -72,11 +73,11 @@ describe('fuzzy-search', () => {
     });
 
     function expectOrder(actual: FuzzySearch.Match<string>[], expected: string[]) {
-        expect(actual.map(result => result.item)).to.be.deep.equal(expected);
+        expect(actual.map(result => result.item)).toEqual(expected);
```

Comment:
I think you have overlooked a few tests from above. It still uses `it` although you have replaced them with `test` everywhere else.

Reply:
Actually Jest does support `it` as a compatibility measure, and in these places `test` was shadowed by the local scope (and the codemod seems to detect it). I've adopted the new syntax anyway. Thanks!